dokka-subprojects/plugin-base-frontend/src/main/types/global.d.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2014-2025 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+/** When Dokka is viewed via iframe, local storage could be inaccessible (see https://github.com/Kotlin/dokka/issues/3323)
+ * The wrapper around local storage to prevent errors in such cases is defined in the plugin-base scripts assets
+ * */
+declare const safeLocalStorage: {
```

Comment:
Here we just need a type declaration because safeLocalStorage is defined in a separate script outside of the ui-kit